### PR TITLE
Bump new interface version to 42 to remove unused flag tileOptimal

### PR DESCRIPTION
**Pull request by mwezdeck**
This flag is supposed to be initialized in VLK XGL
frontend. However, in new code, there is no initialization.

The flag is unused and this is why should  be removed

### CODE REVIEW

**Comment by amdrexu** [comment]
You can only trigger LLPC re-testing by force-push the commit to GitHub.

File: include/vkgcDefs.h [hunk 1]
```diff
@@ -601,9 +601,8 @@ struct SamplerYCbCrConversionMetaData {
       unsigned yChromaOffset : 1; ///< COSITED_EVEN(0) or MIDPOINT(1)
       unsigned xSubSampled : 1;   ///< true(1) or false(0)
       unsigned ySubSampled : 1;   ///< true(1) or false(0)
-      unsigned tileOptimal : 1;   ///< true(1) or false(0)
       unsigned dstSelXYZW : 12;   ///< dst selection Swizzle
-      unsigned undefined : 11;
+      unsigned undefined : 12;
```
Comment (amdrexu) [inline include/vkgcDefs.h:605]: I think your problem is to make compatibility with old codes. So please see VKGC header. A new major version is required and your codes will be
```
#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= X
  unsigned : 1; ///< Unused
#else
  unsigned tileOptimal : 1;   ///< true(1) or false(0)
#endif 
```
All your codes should be gated with version check.

Comment (mwezdeck) [reply include/vkgcDefs.h]: You are right - I changed this in next uploads.

**Comment by mwezdeck** [comment]
Can I kindly ask for review? Thanks.

**Comment by amdrexu** [comment]
I suggest you change the commit tile, says: Bump new interface version to 42 to remove the unused flag tileOptimal. Then, in the description, you can further explain why this is not needed any more.

**Comment by amdvlk-admin** [comment]
### Test summary for commit dd840e3e4b2dc55c4d4288f2e7b5d4ac8bfffa39
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 712f76fee75d69b23a1ea8f6465752c3ccaaf9a2</li>
<li>PAL: dev ea5db60841dab7d067f5010f28a980ef222bdf81</li>
<li>SPVGEN: dev 8dc855026f2502ab3f45dadaf0bb802a57d6ad60</li>
<li>XGL: dev 8d9da0426ba869f5f3da23fd72cfaa669ce99c25</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 8abd9486af1727c8c38afcfef7390d70bb7d7d89</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/43370)</summary><ul><li>Built with branch Vulkan-CTS-1.2.0.2-fetch-sources-fix</li>
<details open><summary>Rhel 8.2, Gfx8</summary><ul>

<li>No Results</li>

</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>No Results</li>

</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 43370/43370 (100.0%)</li>
<li>Failed: 0/43370 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by mwezdeck** [comment]
@amdrexu , could you please merge this pull request?

**Comment by JaxLinAMD** [comment]
retest this please

**Pull request merged**